pkg/media/opus-writer_test.go

```diff
@@ -0,0 +1,163 @@
+package media
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/pions/rtp"
+	"github.com/stretchr/testify/assert"
+)
+
+// PayloadTypes for the default codecs,
+// taken from pions/webrtc/mediaengine.go, to avoid cyclic dependency
+const (
+	defaultPayloadTypeOpus = 111
+	defaultPayloadTypeVP8  = 96
+)
```

Comment:
I'd love to have a way of not hardcoding those values, and use the ones in `mediaengine.go` instead.

Reply:
I would remove the payload restriction, so those numbers are just chosen by the Chromium team. Opus and VP8 are [dynamic](https://en.wikipedia.org/wiki/RTP_audio_video_profile) so nothing you can assert here :(